packages/browser/src/browser/index.ts

```diff
@@ -279,12 +283,21 @@ async function loadAnalytics(
   // needs to be flushed before plugins are registered
   flushPreBuffer(analytics, preInitBuffer)
 
+  const legacyIntegrationSources = plugins.filter(
```

Comment:
This means that we can't support a plugin "creator" function that injects an analytics instance... mmm, this makes me thinks that "integrations" should not be passed in the plugins key.